go/go.mod

```diff
@@ -7,7 +7,7 @@ replace (
 	// https://github.com/spf13/cobra/issues/1091
 	github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2
 
-	github.com/tendermint/tendermint => github.com/oasisprotocol/tendermint v0.33.6-oasis1
+	github.com/tendermint/tendermint => github.com/oasisprotocol/tendermint v0.32.0-dev2.0.20200803174517-82f764f634eb
```

Comment:
Whowa, that's an odd branch to be pointing this at.

Reply:
That's because there is no tag yet, only a branch. Will make a tag now and fix this.

Reply:
Fixed.